future-apps/payroll/test/contracts/Payroll_add_employee.test.js

```diff
@@ -1,5 +1,5 @@
 const { assertRevert } = require('@aragon/test-helpers/assertThrow')
-const { getEvents, getEventArgument } = require('../helpers/events')
+const { getEvents, getEventArgument } = require('../../../../shared/test-helpers/events')
```

Comment:
Could we use the `@aragon/test-helpers/` imports in the Payroll tests too?

Reply:
Sure, I completely ignored this one :)